src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -20,6 +19,10 @@ public class CartPage {
     private final By subscribeBtn_button = By.id("subscribe");
     private final By successAlert_div = By.xpath("//div[@class='alert-success alert']");
 
+    private By removeProduct_btn(String index) {
+        return By.xpath("//a[@class='cart_quantity_delete']["+ index + "]");
```

Comment:
Please change the index value to something more dynamic and describable like the name of the product in your locator.